### ISSUE TITLE
Do not store build / dist / lib files in vcs

### ISSUE THREAD

**Issue by Guria** [zippyui/zippyui.github.io#1]
There is no good reason to store build / dist / lib files in vcs. Everything that could be compiled can be part of npm package and it is not required to duplicate this stuff in git.

@radubrehar 
> `dist` is there for people who whould rather consume that directly as a browser included script, though this is not the preferred approach.
> `lib` really needs to be there, since we expect/want everyone using npm to be able to consume the module, without webpack/babel transpilation. Our entry point in package.json is lib/index.js

**Comment by Guria** [comment]
Both of them must be part of `npm` package but it is not required to store it in vcs. npm package content could be completely different from repository.
You just have to add `build / dist / lib` stuff into `.gitignore` and revert ignoring in `.npmignore`. That's all. Then just run build tasks and `npm publish` as previous.

**Comment by Guria** [comment]
refs:
* [docs](https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package)
* [Some discussion about topic](https://gitter.im/christianalfoni/cerebral?at=56166d5b261e77ba2dbb55a0)
* [SO](http://stackoverflow.com/questions/25124844/should-i-npmignore-my-tests)

**Comment by radubrehar** [comment]
I agree with keeping some things out of the npm package, but we need webpack config files (which we generally keep in `build` to be in the repo.
True, we could keep `dist` in .gitignore, good idea, thanks!
